include/nbl/core/string/StringLiteral.h

```diff
@@ -23,6 +23,6 @@ struct StringLiteral
 }
 
 // for compatibility's sake
-#define NBL_CORE_UNIQUE_STRING_LITERAL_TYPE(STRING_LITERAL) nbl::core::StringLiteral(STRING_LITERAL)
+#define NBL_CORE_UNIQUE_STRING_LITERAL_TYPE(STRING_LITERAL) nbl::core::StringLiteral<sizeof STRING_LITERAL>(STRING_LITERAL)
 
```

Comment:
@kpentaris we changed this quite a bit recently, make sure you've marged latest `master` to yourself